source/client/process_impl.cc

```diff
@@ -92,8 +92,11 @@ class ClusterManagerFactory : public Envoy::Upstream::ProdClusterManagerFactory
   bool prefetch_connections_{};
 };
 
-ProcessImpl::ProcessImpl(const Options& options, Envoy::Event::TimeSystem& time_system)
-    : time_system_(time_system), stats_allocator_(symbol_table_), store_root_(stats_allocator_),
+ProcessImpl::ProcessImpl(const Options& options, Envoy::Event::TimeSystem& time_system,
```

Comment:
Is there any way how we can add test coverage for this change to keep the fixed functionality working?

Reply:
Actually, yes. Lacking C++ mocks for the logging, we can't easily unit-test this issue. 
But I could add an end-to-end integration test for this, see 619c05e